pipeline/frontend/yaml/linter/schema/.woodpecker/test-new-syntax.yml

```diff
@@ -0,0 +1,25 @@
+version: 1
+
+clone:
+  - name: git
+    image: woodpeckerci/plugin-git
+    settings:
+      partial: true
+  - name: testdata
+    image: woodpeckerci/plugin-git
+    settings:
+      remote: https://gitserver/owner/testdata.git
+      path: testdata
```

Comment:
Wasn't aware that you could use this for multiple clone plugins as well

Reply:
yes we are using this quite much, especially for test data, integration tests and sometimes in place of submodules and they also have some drawbacks